src/Cli/Microsoft.DotNet.Cli.Utils/MSBuildForwardingAppWithoutLogging.cs

```diff
@@ -145,7 +145,7 @@ public int ExecuteInProc(string[] arguments)
                 try
                 {
                     // Execute MSBuild in the current process by calling its Main method.
-                    return Microsoft.Build.CommandLine.MSBuildApp.Main(arguments);
```

Comment:
Specifically on this one, "Microsoft.Build" feels like a unit to me, so I'd rather keep the greyed-out part.

Reply:
@Forgind I looked at this file and I have a couple of questions. If you have any answers, I'd appreciate it.
- Is there a reason this was fully-qualified to begin with? Usually, this is only done to avoid naming clashes. I didn't see anything else named `MSBuildApp`.
- This file is wrapped in a `#if NET`. What is the purpose of that? It seems to mess with some of the automatic tooling, at least when trying to "Clean Up Usings" which I attempted to do.

Reply:
> Is there a reason this was fully-qualified to begin with? Usually, this is only done to avoid naming clashes. I didn't see anything else named MSBuildApp.

I didn't write that 😆

The function it's invoking isn't in the SDK at all. It's importing it from MSBuild. VS needs to know where to get the type, and Microsoft.Build is in context, so it can go from there to find MSBuildApp. Alternatively, you can just add a using at the top:
`using Microsoft.Build.CommandLine;`

And then you can skip the qualification and just call MSBuildApp.Main.

The previous version seems to have used reflection, so this was probably chosen as the more similar way that doesn't use reflection. Personally, I like the full qualification in this case so I know it doesn't come from the SDK, but some would argue that I should be able to figure that out by mousing over it or looking at the using. ¯\_(ツ)_/¯

> This file is wrapped in a #if NET. What is the purpose of that? It seems to mess with some of the automatic tooling, at least when trying to "Clean Up Usings" which I attempted to do.

It appears that this one is compiled for both framework and core but only used on framework. It has some things that only exist on core, including this MSBuildApp.Main call (the framework version has no arguments...and also doesn't show up? But I think that's just a using-type issue), and it isn't used on framework, so it may as well only exist on core. The cleaner solution is to just not compile the file at all, but I'm guessing someone added the folder with it and didn't bother adding an Exclude. If you Exclude this file and remove the #if parts, I'll sign off.